akka-http/src/main/scala/akka/http/scaladsl/server/directives/HeaderDirectives.scala

```diff
@@ -199,7 +200,9 @@ trait LowPriorityHeaderMagnetImplicits {
     new HeaderMagnet[T] {
       override def classTag: ClassTag[T] = ClassTag(clazz)
       override def runtimeClass: Class[T] = clazz
-      override def extractPF: PartialFunction[HttpHeader, T] = { case x if runtimeClass.isAssignableFrom(x.getClass) ⇒ x.asInstanceOf[T] }
+      override def extractPF: PartialFunction[HttpHeader, T] = {
```

Comment:
Let's keep the original formatting here and below.

Reply:
Line 187 was on a second line, so the intention here was to give it some consistency across the class and hopefully make it a little more readable. Let me know if it's still not what you want.

Reply:
Ah, I see. Sounds good, then.